cmd/minikube/cmd/config/config.go

```diff
@@ -213,6 +218,79 @@ func configurableFields() string {
 	return strings.Join(fields, "\n")
 }
 
+// AddToConfigArray adds entries to an array in the config file
+func AddToConfigArray(name string, images []string) error {
+	s, err := findSetting(name)
+	if err != nil {
+		return err
+	}
+
+	// Set the values
+	configFile, err := config.ReadConfig()
+	if err != nil {
+		return err
+	}
+	values := configFile[name]
+
+	// Add images to currently existing values in config file
+	if values != nil {
+		for _, v := range values.([]interface{}) {
+			images = append(images, v.(string))
+		}
+	}
+
+	err = s.setArray(configFile, name, images)
+	if err != nil {
+		return err
+	}
+
+	// Write the values
+	return WriteConfig(configFile)
+}
+
+// DeleteFromConfigArray deletes entries in an array in the config file
```

Comment:
maybe map[string]interface{} or map[string]struct{} so then you can just 

`map[key] = struct{}{}` for adding
and `delete(map, key)` for deleting

I think either is fine though.  Using a map makes it more complicated for users to edit the config file themselves (offhand, I'm not sure how the marshalled json would look)

Reply:
Yah I switched it to a map[string]interface{} and it doesn't look too bad in the json so will probably go with that